buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -2501,6 +2502,10 @@ public void testLittleEndianWithExpand() {
 
     private ByteBuf releasedBuffer() {
         ByteBuf buffer = newBuffer(8);
+
+        // Clear the buffer so we are sure the reader and writer indices are 0.
+        // This is important as we may return a slice from newBuffer(...).
+        buffer.clear();
```

Comment:
This is needed as otherwise the test may fail with `IndexOutOfBoundsException` and not `IllegalReferenceCountException` when executed with a slice buffer